plasmapy/formulary/frequencies.py

```diff
@@ -519,3 +520,56 @@ def upper_hybrid_frequency(B: u.T, n_e: u.m**-3) -> u.rad / u.s:
 
 wuh_ = upper_hybrid_frequency
 """Alias to `~plasmapy.formulary.frequencies.upper_hybrid_frequency`."""
+
+
+@validate_quantities(
+    omega_p={"can_be_negative": False},
+    validations_on_return={
+        "units": [u.m**-3],
+        "equivalencies": [
+            (u.F * u.kg * u.rad**2 / (u.C**2 * u.m * u.s**2), u.m**-3)
```

Comment:
Why do you need this equivalencies statement?

Reply:
Astropy uses those units as the return of the calculation for critical density, I assumed it would be cleaner to return the critical density as 1/m^3

Reply:
Oh, yes I guess this accomplishes that. You could also just do that inside this function: 

```python 
n_c = ....
return n_c.to(u.m**-3)
```

I think that would be easier to read, so that's probably my recommendation.

Reply:
astropy doesn't seem to like that
`astropy.units.core.UnitConversionError: 'F kg rad2 / (C2 m s2)' and '1 / m3' (number density) are not convertible`

Reply:
Oh, the problem is the units of radians (since rad isn't a 'real' unit, [it's dimensionless](https://math.stackexchange.com/questions/803955/why-radian-is-dimensionless)). I wish astropy handled this better - it clearly knows they are equivalent because the previous way you had this worked. You could do: 

```python
return (n_c/u.rad**2).to(u.m**-3)
```

Or I guess do it the way you were doing it before, since this isn't great

Reply:
That's definitely a lot cleaner than what I was doing. I'll give that a try

Reply:
Another possibility would be to use the [`dimensionless_angles`](https://docs.plasmapy.org/en/latest/contributing/coding_guide.html#angular-frequencies) equivalency here.

Reply:
Yeah let me give that a try, I thought I saw something about that somewhere

Thanks for the suggestion!